<launch>
	
  <node name="standalone_converter" pkg="costmap_converter" type="standalone_converter" output="screen">
    <param name="converter_plugin" value="costmap_converter::CostmapToDynamicObstacles" />
    <param name="costmap_topic" value="/move_base/local_costmap/costmap" />
    <param name="odom_topic" value="odom" />
  </node>

</launch>
